rotkehlchen/chain/ethereum/manager.py

```diff
@@ -618,20 +620,30 @@ def _ens_lookup(
             ens_resolver_abi.extend(ENS_RESOLVER_ABI_MULTICHAIN_ADDRESS)
             arguments.append(blockchain.ens_coin_type())
 
-        address = self._call_contract(
-            web3=web3,
-            contract_address=to_checksum_address(resolver_addr),
-            abi=ens_resolver_abi,
-            method_name='addr',
-            arguments=arguments,
-        )
+        try:
+            address = self._call_contract(
+                web3=web3,
+                contract_address=deserialize_ethereum_address(resolver_addr),
+                abi=ens_resolver_abi,
+                method_name='addr',
+                arguments=arguments,
+            )
+        except DeserializationError:
+            log.error(f'Error deserializing address {resolver_addr} while doing',
```

Comment:
Sorry :( For how I was reviewing the changes locally I didn't catch that style error. Now I've learned the good way to avoid future mistakes like this one